### ISSUE TITLE
Remove deprecations in twisted.web.error

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5659]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#5659|
|Type|task|
|Created|2012-04-28 16:20:12Z|
|Branch|https://github.com/twisted/twisted/tree/web-error-5659|

The following classes in `twisted.web.error` have been deprecated since 9.0 and can be removed:

 * `ErrorPage`
 * `NoResource`
 * `ForbiddenResource`

<details><summary>Searchable metadata</summary>

```
trac-id__5659 5659
type__task task
reporter__thijs thijs
priority__low low
milestone__ 
branch__branches_web_error_5659 branches/web-error-5659
branch_author__thijs thijs
status__closed closed
resolution__fixed fixed
component__web web
keywords__ 
time__1335630012000000 1335630012000000
changetime__1338136060000000 1338136060000000
version__None None
owner__thijs thijs
cc__thijs cc__jknight
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra commented|
|-|-|

(In [34315]) Branching to 'web-error-5659'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

(In [34316]) Removed deprecated ErrorPage, NoResource, and ForbiddenResource. refs [#5659](https://github.com/twisted/twisted/issues/5659)

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_5"></a>|@therve set owner to @thijstriemstra|
|-|-|

Looks good, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [34463]) Merge web-error-5659: Remove the deprecated aliases for ErrorPage, NoResource and ForbiddenResource in twisted.web.error. 

Author: thijs
Reviewer: therve
Fixes: [#5659](https://github.com/twisted/twisted/issues/5659)
